nixos/modules/services/networking/ssh/sshd.nix

```diff
@@ -160,7 +160,7 @@ in
 
       openFirewall = mkOption {
         type = types.bool;
-        default = true;
+        default = false;
```

Comment:
In this particular case I don't believe the default should change but if it changes anyway then it should be for `stateVersion ≥ 20.03` only.

Reply:
I've made the change anyway.